src/Extensions/TimedScopes/Internal/TimedScopeEventSender.cs

```diff
@@ -35,8 +35,8 @@ public void LogTimedScopeEndEvent(TimedScope scope)
 			string name = activity.OperationName;
 			string correlationId = activity.Id;
 			double durationMs = activity.Duration.TotalMilliseconds;
-			string userHash = activity.GetUserHash(); //Breaking Change: field not set
-			bool isTransaction = activity.IsTransaction(); //Breaking Change: field not set
+			string userHash = activity.GetUserHash(); //TODO: We need add midleware that will set userhash in compliant way and IsTransaction PBI - ADO Office/OC 4016554
```

Comment:
Should we be exposing ADO paths, etc. in a public repo?

Reply:
Not sure, do you think it would be better to have GitHub issues? we tend to act more in ADO for now

Reply:
Switched to GitHub issues